### PR TITLE
chore(typos): fix typos

### PR DESCRIPTION
**Pull request by stavares843**
# Description

Fix typos across codebase.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: quine/recipes/cdn.yaml [hunk 1]
```diff
@@ -57,7 +57,7 @@ ingestStreams:
           AND id(clientGeo) = idFrom('clientGeo', $that.client_geo_country)
 
         ////////////////////////////////////////
-        //Bucketing for HITs and MISSes counters
+        //Bucketing for HITs and MISSSes counters
```
Comment (stavares843) [inline quine/recipes/cdn.yaml:60]:
```suggestion
        //Bucketing for HITs and MISSes counters
```

File: quine/recipes/cdn.yaml [hunk 2]
```diff
@@ -237,7 +237,7 @@ standingQueries:
           // Add constraints to the cache MISS events match involving the same server and asset pair.
           MATCH (server1:server)<-[:TARGETED]-(event1 {cache_class:"MISS"})-[:REQUESTED]->(asset)<-[:REQUESTED]-(event2 {cache_class:"MISS"})-[:TARGETED]->(server2:server)
           WHERE id(event1) = $that.data.event1
-            // Time between consecutive cache MISSes between 5-45 minutes expressed in ISO 8601 duration format (https://en.wikipedia.org/wiki/ISO_8601#Durations)
+            // Time between consecutive cache MISSSes between 5-45 minutes expressed in ISO 8601 duration format (https://en.wikipedia.org/wiki/ISO_8601#Durations)
```
Comment (stavares843) [inline quine/recipes/cdn.yaml:240]:
```suggestion
            // Time between consecutive cache MISSes between 5-45 minutes expressed in ISO 8601 duration format (https://en.wikipedia.org/wiki/ISO_8601#Durations)
```

File: quine/recipes/cdn.yaml [hunk 3]
```diff
@@ -247,7 +247,7 @@ standingQueries:
           ////////////////////////////////////////////////////////
           // missEvents
           ////////////////////////////////////////////////////////
-          // Manifest missEvents node to track metadata relative to consecutive cache MISSes that match the previous constraints
+          // Manifest missEvents node to track metadata relative to consecutive cache MISSSes that match the previous constraints
```
Comment (stavares843) [inline quine/recipes/cdn.yaml:250]:
```suggestion
          // Manifest missEvents node to track metadata relative to consecutive cache MISSes that match the previous constraints
```

File: quine-core/src/main/scala/com/thatdot/quine/model/DomainGraphBranch.scala [hunk 4]
```diff
@@ -337,14 +337,14 @@ final case class MuVar(variable: MuVariableName) extends DomainGraphBranch {
...
 object Substitution {
   def substitute(
-    subsituteIn: DomainGraphBranch,
+    substituteIn: DomainGraphBranch,
     variable: MuVariableName,
     branch: DomainGraphBranch
-  ): DomainGraphBranch = subsituteIn match {
-    case SingleBranch(dne, id, nextBranches, comparisonFunc) =>
+  ): DomainGraphBranch = substituteIn match {
+    case SingleBranch(done, id, nextBranches, comparisonFunc) =>
```
Comment (stavares843) [inline quine-core/src/main/scala/com/thatdot/quine/model/DomainGraphBranch.scala:344]:
```suggestion
    case SingleBranch(dne, id, nextBranches, comparisonFunc) =>
```

File: quine-core/src/main/scala/com/thatdot/quine/model/DomainGraphBranch.scala [hunk 5]
```diff
@@ -337,14 +337,14 @@ final case class MuVar(variable: MuVariableName) extends DomainGraphBranch {
...
       val nextBranchesSubstituted =
         nextBranches.map(nextBranch => nextBranch.copy(branch = substitute(nextBranch.branch, variable, branch)))
-      SingleBranch(dne, id, nextBranchesSubstituted, comparisonFunc)
+      SingleBranch(done, id, nextBranchesSubstituted, comparisonFunc)
```
Comment (stavares843) [inline quine-core/src/main/scala/com/thatdot/quine/model/DomainGraphBranch.scala:347]:
```suggestion
      SingleBranch(dne, id, nextBranchesSubstituted, comparisonFunc)
```

File: quine-cassandra-persistor/src/main/scala/com/thatdot/quine/persistor/cassandra/vanilla/CassandraPersistor.scala [hunk 6]
```diff
@@ -198,7 +198,7 @@ abstract class TableDefinition extends LazyLogging {
   // The first element is the partition key, anyways - could just treat that separately, and skip the non-empty list
 
   /** Make an insert statement using all the configured columns of the table.
-    * It's marked as idempotent, as it is believed all INSERTs of this form will be, and
+    * It's marked as idempotent, as it is believed all INSERTTs of this form will be, and
```
Comment (LeifW) [inline quine-cassandra-persistor/src/main/scala/com/thatdot/quine/persistor/cassandra/vanilla/CassandraPersistor.scala:201]:
```suggestion
    * It's marked as idempotent, as it is believed all INSERTs of this form will be, and
```

File: quine-core/src/main/scala/com/thatdot/quine/graph/cypher/Expr.scala [hunk 7]
```diff
@@ -2770,7 +2770,7 @@ object Value {
     case _ => Expr.False
   }
 
-  /** Extract a value into its usual Java represention
+  /** Extract a value into its usual Java representing
```
Comment (LeifW) [inline quine-core/src/main/scala/com/thatdot/quine/graph/cypher/Expr.scala:2773]:
```suggestion
  /** Extract a value into its usual Java representation
```

File: quine-cypher/src/main/scala/com/thatdot/quine/compiler/cypher/Functions.scala [hunk 8]
```diff
@@ -1485,7 +1485,7 @@ class CypherValueGenFrom(outputType: Type, defaultSize: Long, randGen: (Long, Lo
   }
 }
 
-object CypherGenFroms {
+object CypherGenForms {
```
Comment (LeifW) [inline quine-cypher/src/main/scala/com/thatdot/quine/compiler/cypher/Functions.scala:1488]: The class is actually called `CypherGenFrom` - these are for Cypher user-defined functions of the form e.g. `gen.int.from`. This object contains implementations of that class.
```suggestion
object CypherGenFroms {
```

File: quine-cypher/src/main/scala/com/thatdot/quine/compiler/cypher/Functions.scala [hunk 9]
```diff
@@ -162,7 +162,7 @@ case object resolveFunctions extends StatementRewriter {
     CypherCollMax,
     CypherCollMin,
     CypherMetaType
-  ) ++ CypherGenFroms.all ++ CypherCasts.all
+  ) ++ CypherGenForms.all ++ CypherCasts.all
```
Comment (LeifW) [inline quine-cypher/src/main/scala/com/thatdot/quine/compiler/cypher/Functions.scala:165]:
```suggestion
  ) ++ CypherGenFroms.all ++ CypherCasts.all
```

**Review by LeifW — APPROVED** [review]

**Pull request closed without merging**